app/pages/learn/01_tutorial/03_getting-to-know-the-language/04_classes_objects/01_enums.md

```diff
@@ -0,0 +1,168 @@
+---
+id: lang.classes-objects.enums
+title: Enums
+slug: learn/classes-objects/enums
+type: tutorial-group
+group: classes_objects
+layout: learn/tutorial-group.html
+subheader_select: tutorials
+main_css_id: learn
+toc:
+- What are enums? {intro}
+- Accessing and comparing enums {accessing}
+- Adding members to enums {members}
+- Special methods {functionality}
+- Using enums for singletons {singletons}
+- Abstract methods in enums {abstract}
+description: "Working with enums."
+last_update: 2023-09-28
+author: ["DanielSchmid"]
+---
+<a id="intro">&nbsp;</a>
+## What are enums?
+
+Enums are classes where all instances are known to the compiler.
+They are used for creating types that can only have few possible values.
+
+Enums can be created similar to classes but use the `enum` keyword instead of `class`.
+In the body, there is a list of instances of the enum called enum constants which are seperated by `,`.
+No instances of the enum can be created outside of enum constants.
+
+```java
+public enum DayOfWeek {
+    // enum constant are listed here:
+    MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY
+}
+```
+
+All enums implicitely extend [`java.lang.Enum`](javadoc:Enum) and cannot have any subclasses.
+
+<a id="accessing">&nbsp;</a>
+## Accessing and comparing enums
```

Comment:
Update section title to Accessing, evaluating, and comparing enums.

Add an example of using "compareTo", there are some natural examples like: 

```java
public void compareDayOfWeek(DayOfWeek dayOfWeek){
   int comparison = dayOfWeek.compareTo(DayOfWeek.WEDNESDAY);
   if ( comparison == -1) {
      System.out.println("It's before the middle of the work week.");
   } else if(comparison == 0){
      System.out.println("It's the middle of the work week.");
   } else {
      System.out.println("It's after the middle of the work week.");
   }
}
```

Of course, feel free to change that example as desired, and in the description, mention the declaration order of the enum instances in the enum class is the order that will be used for comparison.

Reply:
I think it makes more sense to put this in the section `Special methods`.

Reply:
I have now added it there. If you think it should still be put in `Accessing and comparing enums`, let me know.

Reply:
Yea, placing it under special methods makes sense, especially with the additional explanation you provided.